<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConnectionTLS_8164A</class>
 <widget class="QWidget" name="ConnectionTLS_8164A">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>268</width>
    <height>149</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Serial to TLS_8164A</string>
  </property>
  <widget class="QComboBox" name="BAUDcomboBox">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>50</y>
     <width>181</width>
     <height>22</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
   <item>
    <property name="text">
     <string>Auto</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>9600</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>19200</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>38400</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="RefreshButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
   <property name="text">
    <string>Refresh</string>
   </property>
  </widget>
  <widget class="QComboBox" name="PORTcomboBox">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>20</y>
     <width>181</width>
     <height>22</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
  </widget>
  <widget class="QLabel" name="BAUDlabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>BAUD</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ConnectButton">
   <property name="geometry">
    <rect>
     <x>152</x>
     <y>90</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Connect</string>
   </property>
   <property name="autoDefault">
    <bool>true</bool>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="PORTlabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>PORT</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>ConnectButton</sender>
   <signal>clicked()</signal>
   <receiver>ConnectionTLS_8164A</receiver>
   <slot>ConnectButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>207</x>
     <y>105</y>
    </hint>
    <hint type="destinationlabel">
     <x>211</x>
     <y>129</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>RefreshButton</sender>
   <signal>clicked()</signal>
   <receiver>ConnectionTLS_8164A</receiver>
   <slot>RefreshButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>73</x>
     <y>108</y>
    </hint>
    <hint type="destinationlabel">
     <x>76</x>
     <y>129</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>ConnectButtonClicked()</slot>
  <slot>RefreshButtonClicked()</slot>
 </slots>
</ui>
